### ISSUE TITLE
Simulation hang/freeze due to integrated shade gas mixture array index problem

### ISSUE THREAD

**Issue by tobiasskov** [NREL/EnergyPlus#6953]
Issue overview
--------------
Running a large parametric study of around 8000 energy models I'm experiencing that some of idfs are unable to complete the simulation - freezing during simulation. 

Given that the only idf files that freezes are some of the models that are containing a construction:complexFenestrationState with a shading layer I think this might be the cause of the hang.

### Details
Some additional details for this issue (if relevant):
 - Win64
 - 8.9.0
 - Unmethours [34060](https://unmethours.com/question/34060/energyplus-freezeshang-during-simulation/)  (confirmed issue by other member)

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added - 
[MSA_bigOffice_WFR40_Orientation112.5.zip](https://github.com/NREL/EnergyPlus/files/2375505/MSA_bigOffice_WFR40_Orientation112.5.zip)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@vidanovic note that @jmarrec says [here](https://unmethours.com/question/34060/energyplus-freezeshang-during-simulation/?comment=34061#post-id-34061) that there is a crash in [TARCOGGasses90.cc#L229](https://github.com/jmarrec/EnergyPlus/blob/771c3799ec7ead41f4f4ca10ab7434a21be64cc5/src/EnergyPlus/TARCOGGasses90.cc#L229).

**Comment by jmarrec** [comment]
The good news is that the file the user sent has only one `Construction:ComplexFenestrationState`. Crash happens in layer 2, which is a `WindowMaterial:ComplexShade`, when it reaches [effectiveLayerCond](https://github.com/jmarrec/EnergyPlus/blob/develop/src/EnergyPlus/ThermalISO15099Calc.cc#L2625:L2640).

Loop on layer, if i=1, it's SPECULAR so nothing to say more. if i=2, it's not SPECULAR, at which point here's the content of the calculated frct1:

```
j=1, frct1(j)=0
j=2, frct1(j)=0.9
j=3, frct1(j)=0.9
j=4, frct1(j)=0.9
j=5, frct1(j)=0
j=6, frct1(j)=0
j=7, frct1(j)=0
j=8, frct1(j)=0
j=9, frct1(j)=0
j=10, frct1(j)=0
```

It calls the [TARCOGGasses90.cc#L229](https://github.com/jmarrec/EnergyPlus/blob/develop/src/EnergyPlus/TARCOGGasses90.cc#L229) with this frct array. If you add this std::cout statement right before that line `std::cout << "frct(" << i << ")=" << frct(i) << "frct(" << j << ")=" << frct(j) << std::endl;`, you get this:

```
frct(1)=0frct(1)=0
frct(1)=0frct(2)=0.9
```

Here's the lldb backtrace:

```
Process 14695 stopped
* thread #1, name = 'energyplus', stop reason = signal SIGFPE: floating point divide by zero
    frame #0: 0x00007ffff628bb63 libenergyplusapi.so.9.0.0`EnergyPlus::TARCOGGasses90::GASSES90(tmean=296.14999999999998, iprop=0x00007ffff7dc0c00, frct=0x00007ffff7dc0c60, pres=100766.66666666667, nmix=2, xwght=0x00007fffffff4eb0, xgcon=0x00007fffffff5570, xgvis=0x00007fffffff55e0, xgcp=0x00007fffffff5650, con=0x00007fffffff42c8, visc=0x00007fffffff42d0, dens=0x00007fffffff42d8, cp=0x00007fffffff42e0, pr=0x00007fffffff43a0, standard=1, nperr=0x00007ffff7d72b08, ErrorMessage="`B�) at TARCOGGasses90.cc:230
   227 	
   228 	                        // calculate the denominator of equation 60
   229 							std::cout << "frct(" << i << ")=" << frct(i) << "frct(" << j << ")=" << frct(j) << std::endl;
-> 230 	                        if (i != j) mukpdwn(i) += phimup / downer * frct(j) / frct(i);
   231 	
   232 	                        // numerator of equation 64, psiterm is the multiplied term in brackets
   233 	                        psiup = pow_2(1.0 + std::sqrt(kprime_i / kprime(j)) / x_pow);


(lldb) bt
* thread #1, name = 'energyplus', stop reason = signal SIGFPE: floating point divide by zero
  * frame #0: 0x00007ffff628bb63 libenergyplusapi.so.9.0.0`EnergyPlus::TARCOGGasses90::GASSES90(tmean=296.14999999999998, iprop=0x00007ffff7dc0c00, frct=0x00007ffff7dc0c60, pres=100766.66666666667, nmix=2, xwght=0x00007fffffff4eb0, xgcon=0x00007fffffff5570, xgvis=0x00007fffffff55e0, xgcp=0x00007fffffff5650, con=0x00007fffffff42c8, visc=0x00007fffffff42d0, dens=0x00007fffffff42d8, cp=0x00007fffffff42e0, pr=0x00007fffffff43a0, standard=1, nperr=0x00007ffff7d72b08, ErrorMessage="`B�) at TARCOGGasses90.cc:230
    frame #1: 0x00007ffff5b3f042 libenergyplusapi.so.9.0.0`EnergyPlus::ThermalISO15099Calc::effectiveLayerCond(nlayer=4, LayerType=ObjexxFCL::Array1A_int @ 0x00007fffffff4d20, scon=const ObjexxFCL::Array1A<double> @ 0x00007fffffff4d70, thick=const ObjexxFCL::Array1A<double> @ 0x00007fffffff4dc0, iprop=ObjexxFCL::Array2A_int @ 0x00007fffffff5490, frct=const ObjexxFCL::Array2A<double> @ 0x00007fffffff5500, nmix=ObjexxFCL::Array1A_int @ 0x00007fffffff4e10, pressure=const ObjexxFCL::Array1A<double> @ 0x00007fffffff4e60, wght=const ObjexxFCL::Array1A<double> @ 0x00007fffffff4eb0, gcon=const ObjexxFCL::Array2A<double> @ 0x00007fffffff5570, gvis=const ObjexxFCL::Array2A<double> @ 0x00007fffffff55e0, gcp=const ObjexxFCL::Array2A<double> @ 0x00007fffffff5650, EffectiveOpenness=const ObjexxFCL::Array1A<double> @ 0x00007fffffff4f00, theta=0x00007fffffff7da0, sconScaled=0x00007fffffff53d0, nperr=0x00007ffff7d72b08, ErrorMessage="`B�) at ThermalISO15099Calc.cc:2645
    frame #2: 0x00007ffff5b3956b libenergyplusapi.so.9.0.0`EnergyPlus::ThermalISO15099Calc::therm1d(nlayer=4, iwd=0, tout=0x00007ffff7d72a68, tind=0x00007ffff7d72a60, wso=0.28309970359262482, wsi=0, VacuumPressure=0, VacuumMaxGapThickness=0, dir=0, ebsky=0x00007fffffff5ed8, Gout=404.27566437166365, trmout=290.58914018661841, trmin=296.16225768584434, ebroom=0x00007fffffff5ee0, Gin=436.19323744193707, tir=0x00007fffffff7a80, rir=0x00007ffff7dbee60, emis=0x00007fffffff7ad0, gap=0x00007fffffff7990, thick=0x00007fffffff79e0, scon=0x00007fffffff7a30, tilt=90, asol=0x00007fffffff7b20, height=2.8000001013278961, heightt=2.8000001013278961, width=3.167671326714363, iprop=0x00007fffffff8660, frct=0x00007fffffff86d0, presure=0x00007fffffff7b70, nmix=0x00007fffffff7c60, wght=0x00007fffffff7bc0, gcon=0x00007fffffff8740, gvis=0x00007fffffff87b0, gcp=0x00007fffffff8820, gama=0x00007fffffff7c10, SupportPillar=0x00007fffffff7cb0, PillarSpacing=0x00007fffffff7d00, PillarRadius=0x00007fffffff7d50, theta=0x00007fffffff7da0, q=0x00007fffffff7df0, qv=0x00007fffffff7e40, flux=0x00007fffffff5ec0, hcin=0x00007ffff7d72a28, hrin=0x00007ffff7d72a20, hcout=0x00007ffff7d72a38, hrout=0x00007ffff7d72a30, hin=0x00007ffff7d72a78, hout=0x00007ffff7d72a80, hcgas=0x00007fffffff7e90, hrgas=0x00007fffffff7ee0, ufactor=0x00007ffff7d72ae8, nperr=0x00007ffff7d72b08, ErrorMessage="`B�, tamb=0x00007ffff7d72b18, troom=0x00007ffff7d72b20, ibc=0x00007fffffff7f30, Atop=0x00007ffff7dc07c0, Abot=0x00007ffff7dc0820, Al=0x00007ffff7dc0880, Ar=0x00007ffff7dc08e0, Ah=0x00007ffff7dc0940, EffectiveOpenness=0x00007fffffff70c0, vvent=0x00007fffffff82f0, tvent=0x00007fffffff8340, LayerType=0x00007fffffff8390, Ra=0x00007fffffff8520, Nu=0x00007fffffff8570, vfreevent=0x00007ffff7dc0760, qcgas=0x00007ffff7dbf1c0, qrgas=0x00007ffff7dbf280, Ebf=0x00007ffff7dc0100, Ebb=0x00007ffff7dc00a0, Rf=0x00007ffff7dc01c0, Rb=0x00007ffff7dc0160, ShadeEmisRatioOut=0x00007ffff7d72b38, ShadeEmisRatioIn=0x00007ffff7d72b40, ShadeHcModifiedOut=0x00007fffffff5e98, ShadeHcModifiedIn=0x00007fffffff5ea0, ThermalMod=0, Debug_mode=0, AchievedErrorTolerance=0x00007fffffff5e90, TotalIndex=0x00007fffffff5e4c, edgeGlCorrFac=1.2510868592668578) at ThermalISO15099Calc.cc:1692
    frame #3: 0x00007ffff5b31e9a libenergyplusapi.so.9.0.0`EnergyPlus::ThermalISO15099Calc::Calc_ISO15099(nlayer=4, iwd=0, tout=0x00007ffff7d72a68, tind=0x00007ffff7d72a60, trmin=0x00007ffff7d72aa8, wso=0.28309970359262482, wsi=0, dir=0, outir=404.27566437166365, isky=3, tsky=283.61546383053053, esky=0x00007ffff7d72ad8, fclr=1, VacuumPressure=0, VacuumMaxGapThickness=0, gap=Array1A<double> @ 0x00007fffffff7990, thick=Array1A<double> @ 0x00007fffffff79e0, scon=Array1A<double> @ 0x00007fffffff7a30, tir=const ObjexxFCL::Array1A<double> @ 0x00007fffffff7a80, emis=const ObjexxFCL::Array1A<double> @ 0x00007fffffff7ad0, totsol=0, tilt=90, asol=const ObjexxFCL::Array1A<double> @ 0x00007fffffff7b20, height=2.8000001013278961, heightt=2.8000001013278961, width=3.167671326714363, presure=const ObjexxFCL::Array1A<double> @ 0x00007fffffff7b70, iprop=ObjexxFCL::Array2A_int @ 0x00007fffffff8660, frct=const ObjexxFCL::Array2A<double> @ 0x00007fffffff86d0, xgcon=const ObjexxFCL::Array2A<double> @ 0x00007fffffff8740, xgvis=const ObjexxFCL::Array2A<double> @ 0x00007fffffff87b0, xgcp=const ObjexxFCL::Array2A<double> @ 0x00007fffffff8820, xwght=const ObjexxFCL::Array1A<double> @ 0x00007fffffff7bc0, gama=const ObjexxFCL::Array1A<double> @ 0x00007fffffff7c10, nmix=ObjexxFCL::Array1A_int @ 0x00007fffffff7c60, SupportPillar=ObjexxFCL::Array1A_int @ 0x00007fffffff7cb0, PillarSpacing=const ObjexxFCL::Array1A<double> @ 0x00007fffffff7d00, PillarRadius=const ObjexxFCL::Array1A<double> @ 0x00007fffffff7d50, theta=Array1A<double> @ 0x00007fffffff7da0, q=Array1A<double> @ 0x00007fffffff7df0, qv=Array1A<double> @ 0x00007fffffff7e40, ufactor=0x00007ffff7d72ae8, sc=0x00007ffff7d72af0, hflux=0x00007ffff7d72af8, hcin=0x00007ffff7d72a28, hcout=0x00007ffff7d72a38, hrin=0x00007ffff7d72a20, hrout=0x00007ffff7d72a30, hin=0x00007ffff7d72a78, hout=0x00007ffff7d72a80, hcgas=Array1A<double> @ 0x00007fffffff7e90, hrgas=Array1A<double> @ 0x00007fffffff7ee0, shgc=0x00007ffff7d72b00, nperr=0x00007ffff7d72b08, ErrorMessage="`B�, shgct=0x00007ffff7d72b10, tamb=0x00007ffff7d72b18, troom=0x00007ffff7d72b20, ibc=ObjexxFCL::Array1A_int @ 0x00007fffffff7f30, Atop=const ObjexxFCL::Array1A<double> @ 0x00007fffffff7f80, Abot=const ObjexxFCL::Array1A<double> @ 0x00007fffffff7fd0, Al=const ObjexxFCL::Array1A<double> @ 0x00007fffffff8020, Ar=const ObjexxFCL::Array1A<double> @ 0x00007fffffff8070, Ah=const ObjexxFCL::Array1A<double> @ 0x00007fffffff80c0, SlatThick=const ObjexxFCL::Array1A<double> @ 0x00007fffffff8110, SlatWidth=const ObjexxFCL::Array1A<double> @ 0x00007fffffff8160, SlatAngle=const ObjexxFCL::Array1A<double> @ 0x00007fffffff81b0, SlatCond=const ObjexxFCL::Array1A<double> @ 0x00007fffffff8200, SlatSpacing=const ObjexxFCL::Array1A<double> @ 0x00007fffffff8250, SlatCurve=const ObjexxFCL::Array1A<double> @ 0x00007fffffff82a0, vvent=const ObjexxFCL::Array1A<double> @ 0x00007fffffff82f0, tvent=const ObjexxFCL::Array1A<double> @ 0x00007fffffff8340, LayerType=ObjexxFCL::Array1A_int @ 0x00007fffffff8390, nslice=ObjexxFCL::Array1A_int @ 0x00007fffffff83e0, LaminateA=const ObjexxFCL::Array1A<double> @ 0x00007fffffff8430, LaminateB=const ObjexxFCL::Array1A<double> @ 0x00007fffffff8480, sumsol=const ObjexxFCL::Array1A<double> @ 0x00007fffffff84d0, Ra=Array1A<double> @ 0x00007fffffff8520, Nu=Array1A<double> @ 0x00007fffffff8570, ThermalMod=0, Debug_mode=0, ShadeEmisRatioOut=0x00007ffff7d72b38, ShadeEmisRatioIn=0x00007ffff7d72b40, ShadeHcRatioOut=0x00007ffff7d72b48, ShadeHcRatioIn=0x00007ffff7d72b50, HcUnshadedOut=0x00007ffff7d72b58, HcUnshadedIn=0x00007ffff7d72b60, Keff=Array1A<double> @ 0x00007fffffff85c0, ShadeGapKeffConv=Array1A<double> @ 0x00007fffffff8610, SDScalar=1, SHGCCalc=0, NumOfIterations=0x00007ffff7d72b68, edgeGlCorrFac=1.2510868592668578) at ThermalISO15099Calc.cc:706
    frame #4: 0x00007ffff5ad5a3c libenergyplusapi.so.9.0.0`EnergyPlus::TARCOGMain::TARCOG90(nlayer=4, iwd=0, tout=0x00007ffff7d72a68, tind=0x00007ffff7d72a60, trmin=0x00007ffff7d72aa8, wso=0.28309970359262482, wsi=0, dir=0, outir=404.27566437166365, isky=3, tsky=283.61546383053053, esky=0x00007ffff7d72ad8, fclr=1, VacuumPressure=0, VacuumMaxGapThickness=0x00007fffffff8ca0, CalcDeflection=0, Pa=100766.66666666667, Pini=0, Tini=-273.14999999999998, gap=Array1A<double> @ 0x00007fffffff99e0, GapDefMax=Array1A<double> @ 0x00007fffffff9a30, thick=Array1A<double> @ 0x00007fffffff9a80, scon=Array1A<double> @ 0x00007fffffff9ad0, YoungsMod=const ObjexxFCL::Array1A<double> @ 0x00007fffffff9b20, PoissonsRat=const ObjexxFCL::Array1A<double> @ 0x00007fffffff9b70, tir=const ObjexxFCL::Array1A<double> @ 0x00007fffffff9bc0, emis=const ObjexxFCL::Array1A<double> @ 0x00007fffffff9c10, totsol=0, tilt=90, asol=const ObjexxFCL::Array1A<double> @ 0x00007fffffff9c60, height=2.8000001013278961, heightt=2.8000001013278961, width=3.167671326714363, presure=const ObjexxFCL::Array1A<double> @ 0x00007fffffff9cb0, iprop=ObjexxFCL::Array2A_int @ 0x00007fffffffa8e0, frct=const ObjexxFCL::Array2A<double> @ 0x00007fffffffa950, xgcon=const ObjexxFCL::Array2A<double> @ 0x00007fffffffa9c0, xgvis=const ObjexxFCL::Array2A<double> @ 0x00007fffffffaa30, xgcp=const ObjexxFCL::Array2A<double> @ 0x00007fffffffaaa0, xwght=const ObjexxFCL::Array1A<double> @ 0x00007fffffff9d00, gama=const ObjexxFCL::Array1A<double> @ 0x00007fffffff9d50, nmix=ObjexxFCL::Array1A_int @ 0x00007fffffff9da0, SupportPillar=ObjexxFCL::Array1A_int @ 0x00007fffffff9df0, PillarSpacing=const ObjexxFCL::Array1A<double> @ 0x00007fffffff9e40, PillarRadius=const ObjexxFCL::Array1A<double> @ 0x00007fffffff9e90, theta=Array1A<double> @ 0x00007fffffff9ee0, LayerDef=Array1A<double> @ 0x00007fffffff9f30, q=Array1A<double> @ 0x00007fffffff9f80, qv=Array1A<double> @ 0x00007fffffff9fd0, ufactor=0x00007ffff7d72ae8, sc=0x00007ffff7d72af0, hflux=0x00007ffff7d72af8, hcin=0x00007ffff7d72a28, hcout=0x00007ffff7d72a38, hrin=0x00007ffff7d72a20, hrout=0x00007ffff7d72a30, hin=0x00007ffff7d72a78, hout=0x00007ffff7d72a80, hcgas=Array1A<double> @ 0x00007fffffffa020, hrgas=Array1A<double> @ 0x00007fffffffa070, shgc=0x00007ffff7d72b00, nperr=0x00007ffff7d72b08, ErrorMessage="`B�, shgct=0x00007ffff7d72b10, tamb=0x00007ffff7d72b18, troom=0x00007ffff7d72b20, ibc=ObjexxFCL::Array1A_int @ 0x00007fffffffa0c0, Atop=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa110, Abot=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa160, Al=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa1b0, Ar=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa200, Ah=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa250, SlatThick=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa2a0, SlatWidth=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa2f0, SlatAngle=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa340, SlatCond=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa390, SlatSpacing=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa3e0, SlatCurve=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa430, vvent=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa480, tvent=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa4d0, LayerType=ObjexxFCL::Array1A_int @ 0x00007fffffffa520, nslice=ObjexxFCL::Array1A_int @ 0x00007fffffffa570, LaminateA=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa5c0, LaminateB=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa610, sumsol=const ObjexxFCL::Array1A<double> @ 0x00007fffffffa660, hg=Array1A<double> @ 0x00007fffffffa6b0, hr=Array1A<double> @ 0x00007fffffffa700, hs=Array1A<double> @ 0x00007fffffffa750, he=0x00007ffff7d72b28, hi=0x00007ffff7d72b30, Ra=Array1A<double> @ 0x00007fffffffa7a0, Nu=Array1A<double> @ 0x00007fffffffa7f0, standard=1, ThermalMod=0, Debug_mode=0, Debug_dir="�#���, Debug_file="\x10$���, win_ID=5, igu_ID=-1, ShadeEmisRatioOut=0x00007ffff7d72b38, ShadeEmisRatioIn=0x00007ffff7d72b40, ShadeHcRatioOut=0x00007ffff7d72b48, ShadeHcRatioIn=0x00007ffff7d72b50, HcUnshadedOut=0x00007ffff7d72b58, HcUnshadedIn=0x00007ffff7d72b60, Keff=Array1A<double> @ 0x00007fffffffa840, ShadeGapKeffConv=Array1A<double> @ 0x00007fffffffa890, SDScalar=1, SHGCCalc=0, NumOfIterations=0x00007ffff7d72b68, edgeGlCorrFac=1.2510868592668578) at TARCOGMain.cc:884
    frame #5: 0x00007ffff4220f64 libenergyplusapi.so.9.0.0`EnergyPlus::WindowComplexManager::CalcComplexWindowThermal(SurfNum=5, ConstrNum=0x00007fffffffba80, HextConvCoeff=1.0603364211713788, SurfInsideTemp=0x00000000037cc2a0, SurfOutsideTemp=0x00000000037ccc60, SurfOutsideEmiss=0x00007fffffffbae0, CalcCondition=0) at WindowComplexManager.cc:3800
    frame #6: 0x00007ffff42a9275 libenergyplusapi.so.9.0.0`EnergyPlus::WindowManager::CalcWindowHeatBalanceInternalRoutines(SurfNum=5, HextConvCoeff=1.0603364211713788, SurfInsideTemp=0x00000000037cc2a0, SurfOutsideTemp=0x00000000037ccc60) at WindowManager.cc:2618
    frame #7: 0x00007ffff42a8fa2 libenergyplusapi.so.9.0.0`EnergyPlus::WindowManager::CalcWindowHeatBalance(SurfNum=5, HextConvCoeff=1.0603364211713788, SurfInsideTemp=0x00000000037cc2a0, SurfOutsideTemp=0x00000000037ccc60) at WindowManager.cc:2459
    frame #8: 0x00007ffff4ec2d60 libenergyplusapi.so.9.0.0`EnergyPlus::HeatBalanceSurfaceManager::CalcHeatBalanceInsideSurf(ZoneToResimulate=ObjexxFCL::Optional_int_const @ 0x00007fffffffd360) at HeatBalanceSurfaceManager.cc:6571
    frame #9: 0x00007ffff4e7926e libenergyplusapi.so.9.0.0`EnergyPlus::HeatBalanceSurfaceManager::ManageSurfaceHeatBalance() at HeatBalanceSurfaceManager.cc:284
    frame #10: 0x00007ffff4dda53e libenergyplusapi.so.9.0.0`EnergyPlus::HeatBalanceManager::ManageHeatBalance() at HeatBalanceManager.cc:360
    frame #11: 0x00007ffff3f80a71 libenergyplusapi.so.9.0.0`EnergyPlus::SimulationManager::ManageSimulation() at SimulationManager.cc:591
    frame #12: 0x00007ffff3d2bb44 libenergyplusapi.so.9.0.0`RunEnergyPlus(filepath="�����) at EnergyPlusPgm.cc:454
    frame #13: 0x00007ffff3d299f4 libenergyplusapi.so.9.0.0`EnergyPlusPgm(filepath="�����) at EnergyPlusPgm.cc:228
    frame #14: 0x00000000004015be energyplus`main(argc=4, argv=0x00007fffffffe508) at main.cc:57
    frame #15: 0x00007ffff1995830 libc.so.6`__libc_start_main(main=(energyplus`main at main.cc:53), argc=4, argv=0x00007fffffffe508, init=<unavailable>, fini=<unavailable>, rtld_fini=<unavailable>, stack_end=0x00007fffffffe4f8) at libc-start.c:291
    frame #16: 0x00000000004014a9 energyplus`_start + 41
```
Floating point exception (core dumped)

**Comment by vidanovic** [comment]
@mjwitte is it too late for bugfix for 9.0.

**Comment by mjwitte** [comment]
@vidanovic Not too late - code will freeze likely sometime early next week.

**Comment by vidanovic** [comment]
Will try to catch some time to fix this one.

**Comment by Myoldmopar** [comment]
Closed via #7243
